Source/src/importers/AnimationImporter.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "assimp/scene.h"
```

Comment:
For external library headers, we use angular braces.
```suggestion
#include <assimp/scene.h>
```